9c-internal/all-in-one/values.yaml

```diff
@@ -0,0 +1,145 @@
+# Default values for 9c-internal cluster.
+# This is a YAML-formatted file.
+
+
+clusterName: "9c-internal-v2"
+logLevel: "debug"
+
+global:
+  image:
+    repository: planetariumhq/ninechronicles-headless
+    tag: "git-7c0815d08cb5d94700918ca4598ae2634c26ac1e"
+
+  appProtocolVersion: "1/54684Ac4ee5B933e72144C4968BEa26056880d71/MEUCIQDFvvGTtUSSRj524xJf0EcxmmU6UaXVDAfp3fsb7P6fkwIgdSicMlCnEUZwsPr+W6VLg+2ReY4+FIRC2vB1Yssz7EU=/ZHU5OnRpbWVzdGFtcHUxMDoyMDIzLTA1LTE5ZQ=="
+  genesisBlockPath: "https://release.nine-chronicles.com/genesis-block-9c-main"
+  trustedAppProtocolVersionSigner: "02529a61b9002ba8f21c858224234af971e962cac9bd7e6b365e71e125c6463478"
+  
+  peerStrings:
+  - "033369e95dbfd970dd9a7b4df31dcf5004d7cfd63289d26cc42bbdd01e25675b6f,9c-internal-tcp.planetarium.dev,31234"
+  
+  iceServers:
+  - "turn://0ed3e48007413e7c2e638f13ddd75ad272c6c507e081bd76a75e4b7adc86c9af:0apejou+ycZFfwtREeXFKdfLj2gCclKzz5ZJ49Cmy6I=@turn-us.planetarium.dev:3478"
+
+  networkType: Internal
+  consensusType: pbft
+  
+  useExternalSecret: true
+
+snapshot:
+  resetSnapshot: true
+  slackChannel: "9c-internal"
+  image: "planetariumhq/ninechronicles-snapshot:git-fc194ebdc47ec4ebe14f97f5ea01ff97b0a276d6"
+
+# if you want to delete PVC with the volume provisioned together, set this value "Delete"
+volumeReclaimPolicy: "Retain"
+
+seed:
+  count: 1
+  useTurnServer: false
+  image:
+    repository: planetariumhq/libplanet-seed
+    pullPolicy: Always # Overrides the image tag whose default is the chart appVersion.
+    tag: "git-22bed8ef41ab09258e6404462ec8e07cd038d089"
+
+  hosts:
+  - "9c-main-tcp-seed-1.planetarium.dev"
+
+validator:
+  count: 4
+  image:
+    repository: planetariumhq/ninechronicles-headless
+    pullPolicy: Always
+    tag: revert-gas-limit
+
+  consensusSeedStrings:
+  - 033369e95dbfd970dd9a7b4df31dcf5004d7cfd63289d26cc42bbdd01e25675b6f,tcp-seed-1,31235
+
+  hosts:
+  - "9c-internal-validator-1.nine-chronicles.com"
+  - "9c-internal-validator-2.nine-chronicles.com"
+  - "9c-internal-validator-3.nine-chronicles.com"
+  - "9c-internal-validator-4.nine-chronicles.com"
+
+  storage:
+    data: 500Gi
+
+  resources:
+    requests:
+      cpu: 1
+      memory: 12Gi
+
+remoteHeadless:
+  image:
+    repository: planetariumhq/ninechronicles-headless
+    pullPolicy: Always
+    tag: revert-gas-limit
+
+  hosts:
+  - "9c-internal-rpc-1.nine-chronicles.com"
+
+  ports:
+    headless: 31234
+    graphql: 80
+    rpc: 31238
+
+  storage:
+    data: 500Gi
+
+  resources:
+    requests:
+      cpu: 1
+      memory: 12Gi
+
+  nodeSelector:
+    beta.kubernetes.io/os: linux
```

Comment:
i think we should checkout every nodeSelectors, validator also

Reply:
Can you explain more?

Reply:
https://github.com/planetarium/9c-infra/blob/main/9c-internal/chart/templates/remote-headless.yaml#L163-L164
as we can see above, we need more nodeSelector config for remote-headless, validator, etc.
previously these are injected directly to chart template, not by value